canonicalwebteam/discourse/parsers/base_parsers.py

```diff
@@ -0,0 +1,223 @@
+# Standard library
+import re
+
+# Packages
+from bs4 import BeautifulSoup
+from urllib.parse import urlparse
+
+
+TOPIC_URL_MATCH = re.compile(
+    r"(?:/t)?(?:/(?P<slug>[^/]+))?/(?P<topic_id>\d+)(?:/\d+)?"
+)
+
+
+class BaseParser(object):
```

Comment:
I think it will be better without the s at the end, sorry for being picky :/ @carkod

Reply:
Done